### PR TITLE
[FIX][web_responsive] Long-named multicompanies nice in XS

### PR DESCRIPTION
**Pull request by yajo**
Before this patch:
captura el 2017-08-08 a las 14 00 20

After it:
captura el 2017-08-08 a las 14 00 25
captura el 2017-08-08 a las 14 17 37

@Tecnativa

### CODE REVIEW

File: web_responsive/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "name": "Web Responsive",
     "summary": "It provides a mobile compliant interface for Odoo Community "
                "web",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.0.1",
```
Comment (pedrobaeza) [inline web_responsive/__manifest__.py]: This requires to update the module, so put `10.0.1.1.0`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please hide the building icon on large displays.

File: web_responsive/__manifest__.py (hunk 1 above)
Comment (yajo) [reply web_responsive/__manifest__.py]: Are you sure? I think that's not needed to update a client-side qweb template, isn't it?

Comment (pedrobaeza) [reply web_responsive/__manifest__.py]: Well, I'm not sure either, but anyway, it doesn't hurt

**Comment by pedrobaeza** [comment]
What about the other request?

**Comment by yajo** [comment]
Ah sorry, I think it's more consistent if we display the same icon in all sizes. This way when using XS viewport you are already familiar with that icon and its meaning.

**Comment by pedrobaeza** [comment]
Well, there are other elements that change notably when they are in small size, and it's very easy to discover for what is the building with the dropdown arrow, so I prefer to keep big screen without that icon.

**Comment by yajo** [comment]
I feel that's anti-UX, but will do.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@lasley @chienandalu please review

**Review by chienandalu — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
:+1: Looks nice

**Comment by lasley** [comment]
Bit late to the party, but 👍 !

**Pull request merged**